packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2414,6 +2414,50 @@ void main() {
 
     expect(find.byType(CupertinoPickerDefaultSelectionOverlay), findsExactly(4));
   });
+
+  testWidgets('CupertinoDatePicker getColumnWidth', (WidgetTester tester) async {
+    const List<String> testWords = <String>[
+      'aaaaaii',    // Narrow characters - 7 characters
```

Comment:
All these characters should have the exact same advance: https://github.com/flutter/flutter/blob/master/docs/contributing/testing/Flutter-Test-Fonts.md#glyph-mapping

So I would expect 'aaaaaii' to have a larger width than `WWWWWW`.

Reply:
Consider using characters that are mapped to glyphs with <= 1 EM advance.

Reply:
Fixed. My test was giving false positive.